google/cloud/bigtable/mutation_batcher.h

```diff
@@ -161,7 +155,6 @@ class MutationBatcher {
    */
   std::pair<future<void>, future<Status>> AsyncApply(CompletionQueue& cq,
                                                      SingleRowMutation mut);
-
```

Comment:
I'd probably restore this separating line.

Reply:
Whoops. Thanks for catching that